client/src/components/Logo/Logo.test.js

```diff
@@ -1,16 +0,0 @@
-import React from 'react'
-import { shallow, configure } from 'enzyme'
-import Logo from './Logo.js'
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
-
-// We will describe a block of tests
-describe('Logo component', () => {
```

Comment:
Why was this file deleted?

Reply:
I super don't remember/have no idea what happened with it